### PR TITLE
Scala and Ruby article in german

### PR DESCRIPTION
**Pull request by coastalchief**
Translated scala article into german and added paragraph on traits.
Translated ruby article

### CODE REVIEW

**Comment by coastalchief** [comment]
Changed headings in ruby and scala file.

File: de-de/ruby-de.html.markdown [hunk 1]
```diff
@@ -0,0 +1,613 @@
...
+  - ["Tristan Hume", "http://thume.ca/"]
+  - ["Nick LaMuro", "https://github.com/NickLaMuro"]
+  - ["Marcos Brizeno", "http://www.about.me/marcosbrizeno"]
+  - ["Ariel Krakowski", "http://www.learneroo.com"]
+  - ["Dzianis Dashkevich", "https://github.com/dskecse"]
+  - ["Levi Bostian", "https://github.com/levibostian"]
+  - ["Rahil Momin", "https://github.com/iamrahil"]
+translators:
+  - ["Christian Albrecht", "https://github.com/coastalchief"]
+filename: ruby-de.html.markdown
```
Comment (adambard) [inline de-de/ruby-de.html.markdown]: The filename is used for the generated code download. So, maybe `ruby-de.rb`?

Comment (adambard) [inline de-de/ruby-de.html.markdown]: Ah, nevermind, I'll just do it. Thanks!

**Comment by adambard** [comment]
Danke!

**Comment by coastalchief** [comment]
Thanks!

**Pull request merged**